app/konnect/runtime-manager/kic.md

```diff
@@ -34,32 +32,48 @@ To associate your KIC runtime instances with {{site.konnect_short_name}}, use th
 In {{site.konnect_short_name}}, navigate to {% konnect_icon runtimes %} **[Runtime Manager](https://cloud.konghq.com/runtime-manager)**, then click **New Runtime Group** > **Kong Ingress Controller**.
 
 {:.note}
-> **Note**: KIC OSS users can connect to {{site.konnect_short_name}}’s Free tier, while KIC Enterprise users can connect to {{site.konnect_short_name}}’s Enterprise tier.
+> **Note**: KIC OSS users can connect to {{site.konnect_short_name}}’s Free tier, while KIC Enterprise users can connect to {{site.konnect_short_name}}’s Enterprise tier. To migrate from {{site.ce_product_name}} to {{site.base_gateway}} Enterprise, see [Using Kong Enterprise](/kubernetes-ingress-controller/latest/guides/choose-gateway-image/) in the Kong Ingress Controller documentation.
```

Comment:
The Free tier connects to a Kong Gateway in Free mode by default. Customers can connect their OSS Gateways in the Free tier. Is this an important enough distinction to make? Thoughts?

Reply:
Additionally, Konnect does not support analytics for OSS Gateways yet. Free tier users can use Kong Gateway in Free mode to get analytics.